src/components/partials/tables/FTable.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+    <div class="q-px-none q-py-sm">
+        <div
+            v-if="pageHeader"
+            class="row"
+        >
+            <div class="flex col-xs-12 col-md-6 q-pa-md">
+                <slot name="header-left">
+                    <span
+                        v-if="headerHelpText"
+                        class="text-grey-8"
+                        v-text="headerHelpText"/>
+
+                    <q-breadcrumbs
+                        v-else-if="breadcrumbs.length > 0">
+                        <template v-slot:separator>
+                            <q-icon
+                                size="1.3em"
+                                name="chevron_right"
+                                color="grey-7"
+                            />
+                        </template>
+                        <q-breadcrumbs-el
+                            v-for="breadcrumb in breadcrumbs"
+                            :label="breadcrumb.label"
+                            :key="breadcrumb.label"
+                            :to="breadcrumb.to"
+                            class="text-grey-7"
+                        />
+                    </q-breadcrumbs>
+
+                </slot>
+            </div>
+            <div class="col-xs-12 col-md-6">
+                <div class="float-right">
+                    <slot name="header-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row lt-md">
+            <div class="col-xs-12 border-bottom-grey">
+                <div class="float-right q-mb-sm q-mx-sm">
+                    <slot name="control-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div
+                :class="tabIndicatorColor === 'transparent' ? 'q-pa-sm' : ''"
+                class="col-xs-12 col-md-6 border-bottom-grey "
+            >
+                <slot name="control-left">
+
+                    <div
+                        v-if="tableTitle"
+                        class="text-h5 q-pa-md"
+                        v-text="tableTitle"
+                    />
+                    <q-tabs
+                        v-else
+                        v-model="selectedTab"
+                        :indicator-color="tabIndicatorColor"
+                        :active-color="tabIndicatorColor === 'transparent' ? 'white' : 'primary'"
+                        :class="tabIndicatorColor === 'transparent' ? 'pa-xs' : ''"
+                        inline-label
+                        no-caps
+                        stretch
+                        align="left"
+                    >
+                        <q-tab
+                            v-for="tab in tabs"
+                            :key="tab.name"
+                            v-bind="tab"
+                            :name="tab.name"
+                            :class="tab.name === selectedTab && isDark ? 'bg-grey-9 border-radius-3' : ''"
```

Comment:
The dark theme should be handled at the theme/css level, not at the component level?

Reply:
Now it's handled through CSS only, but I added CSS on `FTable` component.